packages/relay/src/lib/services/debugService/IDebugService.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+export interface IDebugService {
+  debug_traceTransaction: (
+    transactionHash: string,
+    tracer: string,
```

Comment:
would not make sense to change `tracer` type, from string to enum?

with the following 2 options:
1. callTracer
2. prestateTracer

https://www.quicknode.com/docs/ethereum/debug_traceTransaction

Reply:
Done :)